end-to-end-tests/cypress.json

```diff
@@ -1,4 +1,5 @@
 {
   "chromeWebSecurity": false,
-  "projectId": "jd9h7w"
+  "projectId": "jd9h7w",
+  "video": false
```

Comment:
Can't remember where we got to with this - e.g. does it work in some cases, if so might be worth leaving in but can have a chat about it later